Tasks/NuGetInstaller/VsoNuGetHelper.ps1

```diff
@@ -0,0 +1,157 @@
+Add-Type -AssemblyName System.Security
+
+$nuGetTempDirectory = Join-Path $Env:AGENT_BUILDDIRECTORY "NuGet\"
+$tempNuGetConfigPath = Join-Path $nuGetTempDirectory "newNuGet.config"
+
+function SaveTempNuGetConfig
+{
```

Comment:
You should add [cmdletbinding()] to every function. It needs to be the very first statement in the function. E.g.

function MyVerb-MyNoun {
  [cmdletbinding()]
  param( [...] )
  [...]
}